packages/app-root/src/components/PageContextProviders.js

```diff
@@ -17,27 +17,33 @@ const GlobalStyle = createGlobalStyle`
 /**
   Context for every page:
   - global page styles.
-  - Zooniverse Grommet theme.
+  - Zooniverse Grommet theme and mode.
   - Panoptes auth (user account and admin mode.)
 */
 export default function PageContextProviders({ children }) {
   const { data: user, error, isLoading } = usePanoptesUser()
   const { adminMode, toggleAdmin } = useAdminMode(user)
   const authContext = { adminMode, error, isLoading, toggleAdmin, user }
 
+  const { themeMode, toggleTheme } = useThemeMode()
+  console.log('PAGE CONTEXT', themeMode)
```

Comment:
I've now run into a problem where passing `themeMode` from `useThemeMode()` to the Grommet provider in PageContextProviders isn't working as expected. Grommet won't re-render with dark mode in a new tab. For instance, if I choose dark mode on app-root https://local.zooniverse.org:3000, close and re-open the browser tab, this `themeMode` is `'dark'`, but doesn't change the overall Grommet theme as I'd expect. 

Only the components that I have waiting for `hasMounted` recognize that `useThemeMode` is returning `themeMode = 'dark'`. I'll have to think about this for a bit before ready for review..